invenio_search_ui/assets/js/invenio_react_search_ui/Results.js

```diff
@@ -0,0 +1,146 @@
+    const metadata = result.metadata;
+    return (
+      <Item key={index} href={`#`}>
+        <Item.Image
+          size="small"
+          src={result.imageSrc || "https://placehold.it/200"}
+        />
+        <Item.Content>
+          <Item.Header>{metadata.title}</Item.Header>
+          <Item.Description>
+            {_truncate(metadata.keywords.join(","), { length: 200 })}
+          </Item.Description>
+        </Item.Content>
+      </Item>
+    );
+  };
+
+  renderResultsGridItem(result, index) {
+    const metadata = result.metadata;
+    return (
+      <Card fluid key={index} href={`#`}>
+        <Image src={result.imageSrc || "https://placehold.it/200"} />
+        <Card.Content>
+          <Card.Header>{metadata.title}</Card.Header>
+          <Card.Description>
+            {_truncate(metadata.keywords.join(","), { length: 200 })}
+          </Card.Description>
+        </Card.Content>
+      </Card>
+    );
+  }
+
+  renderCount(totalResults) {
+    return (
+      <Label size="mini" color="blue">
+        {totalResults}
+      </Label>
+    );
+  }
+
+  render() {
+    const { total } = this.props.currentResultsState.data;
+    const CustomResultsListCmp = () => (
+      <ResultsList renderListItem={this.renderResultsListItem} />
+    );
+    const CustomResultsGridCmp = () => (
+      <ResultsGrid renderGridItem={this.renderResultsGridItem} />
+    );
+    return total ? (
+      <>
+        <Grid relaxed>
+          <Grid.Row>
+            <Grid.Column width={16}>
+              <ActiveFilters />
+            </Grid.Column>
+          </Grid.Row>
+          <Grid.Row verticalAlign="middle">
+            <Grid.Column width={7}>
+              <Count renderElement={this.renderCount} />
+              <SpanWithMargin text="result(s) found" />
+              <br />
+              <SpanWithMargin text="Show" margin="right" />
+              <ResultsPerPage
+                values={this.resultsPerPageValues}
+                defaultValue={10}
+              />
+              <SpanWithMargin text="results per page" />
+            </Grid.Column>
+            <Grid.Column width={7} textAlign="right">
+              <SpanWithMargin text="Sort by" />
+              <Sort values={this.sortValues} />
+            </Grid.Column>
+              <Grid.Column width={2} textAlign="right">
+              <LayoutSwitcher defaultLayout="list" />
+            </Grid.Column>
+          </Grid.Row>
+          <Grid.Row>
+            <Grid.Column>
+              <ResultsMultiLayout
+                resultsListCmp={CustomResultsListCmp}
+                resultsGridCmp={CustomResultsGridCmp}
+              />
+            </Grid.Column>
+          </Grid.Row>
+          <Grid.Row verticalAlign="middle" textAlign="center">
+            <Grid.Column>
+              <Pagination />
+            </Grid.Column>
+          </Grid.Row>
+        </Grid>
+      </>
+    ) : null;
+  }
+}
+
+Results.propTypes = {};
```

Comment:
Please make sure that at some point (probably later?) you fill in all propTypes and defaultProps :)